src/modules/emote_menu/utils/twitch-emotes.js

```diff
@@ -123,7 +162,10 @@ export async function loadTwitchEmotes() {
           '2x': TWITCH_EMOTE_CDN(emoteId, '2.0', isDark),
           '4x': TWITCH_EMOTE_CDN(emoteId, '3.0', isDark),
         },
-        metadata: {type},
+        metadata: {
+          type,
+          locked,
```

Comment:
locked here should be a predicate checking if the user is subscribed or not, or following or not (depends on emote type) when rendering.